icu-filters/icudt_browser.json

```diff
@@ -156,7 +156,7 @@
             "ms_BN",
             "ms_MY",
             "ms_SG",
-            "nb-NO",
+            "nb_NO",
```

Comment:
@lewing @radical should we add this culture for wasm?

Reply:
I'd prefer if we did so we don't have this once culture difference between iOS and wasm.